app/controllers/Application.scala

```diff
@@ -101,11 +101,19 @@ class Application(implicit val wsClient: WSClient, val db: MetricsDB) extends Co
     }
   }
 
-  def getGroupedWordCounts() = APIAuthAction { req =>
+  def getArticlesGroupedByLength() = APIAuthAction { req =>
     APIResponse {
       for {
-        groupedWordCounts <- db.getGroupedWordCounts(Filters(req.queryString))
-      } yield groupedWordCounts
+        groupedByArticleLength <- db.getArticlesGroupedByLengthBounds()(Filters(req.queryString))
```

Comment:
Could pass a parameter here as well for clarity. Instead of passing in a boolean to function, you could pass in the name of the field as they are of equal important. The parameter could be a case object so you could do something like `getArticlesGroupedByLengthBounds(ArticleLength.CommissionedLength)` and `getArticlesGroupedByLengthBounds(ArticleLength.FinalLength)`

Reply:
Yes I think this is a much better approach